src/components/shared/CallNodeContextMenu.js

```diff
@@ -688,39 +683,28 @@ export const CallNodeContextMenu = explicitConnect<
   component: CallNodeContextMenuImpl,
 });
 
-function TransformMenuItem(props: {|
+function MenuItemWithShortcut(props: {|
   +children: React.Node,
   +onClick: (event: SyntheticEvent<>, data: { type: string }) => void,
-  +transform: string,
   +shortcut: string,
-  +icon: string,
-  +title: string,
+  +data: { type: string },
 |}) {
   return (
-    <MenuItem
-      onClick={props.onClick}
-      data={{ type: props.transform }}
-      attributes={{ title: oneLine`${props.title}` }}
-    >
-      <span
-        className={`react-contextmenu-icon callNodeContextMenuIcon${props.icon}`}
-      />
+    <MenuItem onClick={props.onClick} data={{ type: props.data.type }}>
       <div className="react-contextmenu-item-content">{props.children}</div>
       <kbd className="callNodeContextMenuShortcut">{props.shortcut}</kbd>
     </MenuItem>
   );
 }
 
-function MenuItemWithShortcut(props: {|
+function DivWithTitle(props: {|
+  +className: string,
```

Comment:
I feel like the className could be optional, what do you think?

Reply:
Sounds good to me, changed it.